client/landing/stepper/declarative-flow/internals/steps-repository/__user/index.tsx

```diff
@@ -57,52 +59,64 @@ const UserStepComponent: Step = function UserStep( {
 		redirectTo,
 	} );
 
+	const locale = useFlowLocale();
+	const shouldRenderLocaleSuggestions = ! isLoggedIn; // not sure if this is necessary. It's used in Classic login
```

Comment:
Probably because for logged-in users, we respect the user language settings.

Reply:
I think that makes sense. Updated the comment with what you wrote too :-)